DESCRIPTION

```diff
@@ -9,19 +9,20 @@ Description: Complex machine learning models are often hard to interpret. Howeve
   described in Aas, Jullum and Løland (2019) <arXiv:1903.10464>, which accounts for any feature 
   dependence, and thereby produces more accurate estimates of the true Shapley values.
 Authors@R: c(
-    person("Nikolai", "Sellereite", email = "Nikolai.Sellereite@nr.no", role = "cre"),
-    person("Martin", "Jullum", email = "Martin.Jullum@nr.no", role = "aut"),  
+    person("Nikolai", "Sellereite", email = "nikolaisellereite@gmail.com", role = c("cre", "aut"), comment = c(ORCID = "0000-0002-4671-0337")),
+    person("Martin", "Jullum", email = "Martin.Jullum@nr.no", role = "aut", comment = c(ORCID = "0000-0003-3908-5155")),  
```

Comment:
Maybe I should also be added here

Reply:
Yes! Do you have an ORCID (no need to create one if you don't have one)?

Reply:
No I don't think so...